src/mechanics/timeIntegration/RungeKuttaBase.h

```diff
@@ -38,7 +38,22 @@ class RungeKuttaBase : public TimeIntegrationBase
     //! @brief ... return, if time (e.g. for the calculation of external loads) has changed
     virtual bool HasTimeChanged(int rStage) const = 0;
 
+
+    // temporary fix
+    //! @brief sets the  time step for the time integration procedure (initial value)
+    virtual void SetTimeStep(double rTimeStep) override
```

Comment:
What is the temporary fix for? Why did mTime and mTimeStep moved from TimeIntegrationBase?